docker/Dockerfile.keygen

```diff
@@ -1,10 +1,10 @@
 FROM eclipse-temurin:17-alpine
-MAINTAINER radixdlt <devops@radixdlt.com>
+LABEL org.opencontainers.image.authors="devops@radixdlt.com"
```

Comment:
I'm not sure how this build works - I imagine we might wish to update this to use the new docker file - probably in a follow up PR

Reply:
I also think about follow up PR. Because I have no idea about the build/publish process of this yet.